internal/pkg/cli/task_run.go

```diff
@@ -445,6 +452,33 @@ func (o *runTaskOpts) runTask() ([]*task.Task, error) {
 	return tasks, nil
 }
 
+func (o *runTaskOpts) showPublicIPs(tasks []*task.Task) error {
+	publicIPs := make(map[string]string)
+	for _, t := range tasks {
+		if t.ENI == "" {
+			continue
+		}
+		ip, err := o.publicIPGetter.PublicIP(t.ENI) // We will just not show the ip address if an error occurs.
+		if err == nil {
+			publicIPs[t.TaskARN] = ip
+		}
+	}
+
+	if len(publicIPs) > 0 {
```

Comment:
nit: (flipping the if-statement to reduce nesting)
```suggestion
	if len(publicIPs) == 0 {
		return
	}
	log.Info("...
```